File: springboot-deployment/src/main/java/be/insaneprogramming/cleanarch/Runner.java
package be.insaneprogramming.cleanarch;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.builder.SpringApplicationBuilder;
import org.springframework.boot.web.support.SpringBootServletInitializer;
import org.springframework.transaction.annotation.EnableTransactionManagement;

@SpringBootApplication
@EnableTransactionManagement
public class Runner extends SpringBootServletInitializer {

	@Override
	protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
		return application.sources(Runner.class);
	}

	public static void main(String[] args) {
		SpringApplication.run(Runner.class, args);
	}
}

File: springboot-deployment/src/main/kotlin/be/insaneprogramming/cleanarch/Runner.kt
package be.insaneprogramming.cleanarch

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
open class Runner {
    companion object {
        @JvmStatic fun main(args: Array<String>) {
            SpringApplication.run(Runner::class.java, *args)
        }
    }
}
